lib/src/dialog/Dialog.tsx

```diff
@@ -143,9 +144,8 @@ const CloseIconAction = styled.button`
   &:active {
     background-color: #cccccc;
   }
-  svg {
-    width: ${(props) => props.theme.closeIconWidth};
-    height: ${(props) => props.theme.closeIconHeight};
+  span {
+    font-size: ${(props) => props.theme.closeIconHeight};
   }
```

Comment:
I would rename this token to `closeIconSize` and remove `closeIconWidth` and `closeIconHeight`, since they are not used now.